docs/content.zh/docs/core-concept/schema-evolution.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Schema Evolution"
+weight: 7
+type: docs
+aliases:
+  - /core-concept/schema-evolution/
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# 定义
+
+**Schema Evolution** 功能可以用于将上游的 DDL 变更事件同步到下游，例如创建新表、添加新列、重命名列或更改列类型、删除列、截断和删除表等。
+
+## 参数
+
+Schema Evolution 的行为可以通过配置以下参数来设定：
+
+```yaml
+pipeline:
+  schema.change.behavior: evolve
+```
+
+`schema.change.behavior` 是一个枚举类型，可以被设定为 `exception`、`evolve`、`try_evolve`、`lenient`、或 `ignore`。
+
+## Schema Evolution 行为
+
+### Exception 模式
+
+在此模式下，所有结构变更行为均不被允许。
+一旦收到表结构变更事件，`SchemaOperator` 就会抛出异常。
+当您的下游接收器不能处理任何架构更改时，可以使用此模式。
+
+### Evolve 模式
+
+在此模式下，`SchemaOperator` 会将所有上游架构更改事件应用于下游接收器。
+如果尝试失败，则会从 `SchemaRegistry` 抛出异常并触发全局的故障重启。
+
+### TryEvolve 模式
+
+在此模式下，架构运算符还将尝试将上游架构更改事件应用于下游接收器。
+但是，如果下游接收器不支持特定的架构更改事件并报告失败，
+`SchemaOperator` 会容忍这一事件，并且在出现上下游表结构差异的情况下，尝试转换所有后续数据记录。
+
+> 警告：此类数据转换和转换不能保证无损。某些数据类型不兼容的字段可能会丢失。
+
+### Lenient 模式
+
+在此模式下，架构操作员将在转换所有上游架构更改事件后将其转换为下游接收器，以确保不会丢失任何数据。
+例如，`AlterColumnTypeEvent` 将被转换为两个单独的架构更改事件 `RenameColumnEvent` 和 `AddColumnEvent`：
+保留上一列（具有更改前的类型），并添加一个新列（具有新类型）。
+
+这是默认的架构演变行为。
+
+> 注意：在此模式下，`TruncateTableEvent` 和 `DropTableEvent` 默认不会被发送到下游，以避免意外的数据丢失。这一行为可以通过配置 [Per-Event Type Control](#per-event-type-control) 调整。
```

Comment:
Hi, @yuxiqian, I found a dead line in https://nightlies.apache.org/flink/flink-cdc-docs-master/zh/docs/core-concept/schema-evolution/#lenient-%e6%a8%a1%e5%bc%8f

Reply:
Thanks for pointing it out! Will raise another PR to clean-up format issues in docs later.